addons/website_sale/models/res_config_settings.py

```diff
@@ -69,31 +69,24 @@ def _compute_terms_url(self):
         for record in self:
             record.terms_url = '%s/terms' % record.website_id.get_base_url()
 
-    @api.model
-    def get_values(self):
-        res = super(ResConfigSettings, self).get_values()
-
-        sale_delivery_settings = 'none'
-        if self.env['ir.module.module'].search([('name', '=', 'delivery')], limit=1).state in ('installed', 'to install', 'to upgrade'):
-            sale_delivery_settings = 'internal'
-            if self.env['ir.module.module'].search([('name', '=', 'website_sale_delivery')], limit=1).state in ('installed', 'to install', 'to upgrade'):
-                sale_delivery_settings = 'website'
-
-        res.update(
-            sale_delivery_settings=sale_delivery_settings,
```

Comment:
Why is this logic of delivery settings disappearing here?  How is it related to the current issue/fix ?